exercise-core/src/main/java/com/examples/attsw/exercise/model/repository/mongo/MongoDatabaseWrapper.java

```diff
@@ -0,0 +1,55 @@
+package com.examples.attsw.exercise.model.repository.mongo;
+
+import java.net.UnknownHostException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import org.jongo.Jongo;
+import org.jongo.MongoCollection;
+
+import com.mongodb.BasicDBObject;
+import com.mongodb.DB;
+import com.mongodb.DBCollection;
+import com.mongodb.DBCursor;
+import com.mongodb.DBObject;
+import com.mongodb.MongoClient;
+public class MongoDatabaseWrapper implements Database {
```

Comment:
The issue required to implement another class, NOT MongoDatabaseWrapper (see the previous comment)